src/main/java/emu/grasscutter/command/commands/WeatherCommand.java

```diff
@@ -11,29 +11,30 @@
 
 import static emu.grasscutter.utils.Language.translate;
 
-@Command(label = "weather", usage = "weather <climate type(weatherId)> <weather type(climateId)>", aliases = {"w"}, permission = "player.weather", permissionTargeted = "player.weather.others", description = "commands.weather.description")
+@Command(label = "weather", usage = "weather <climate> <weather>", aliases = {"w"}, permission = "player.weather", permissionTargeted = "player.weather.others", description = "commands.weather.description")
 public final class WeatherCommand implements CommandHandler {
 
     @Override
     public void execute(Player sender, Player targetPlayer, List<String> args) {
         int weatherId = 0;
         int climateId = 1;
         switch (args.size()) {
-            case 2:
+            case 0:
```

Comment:
This won't actually work, you're trying to pull index-0 and index-1 from a size-0 array, and index-1 from a size-1 array.